### ISSUE TITLE
codegpt login with oauth not possible due to redirects

### ISSUE THREAD

**Issue by austinfarmer11** [JudiniLabs/code-gpt-docs#387]
I have been struggling to login to codegpt.  I continue to get an error after installing the extension and trying to sign up.  The error involves redirect URL problems.  When I attempt to sign in or sign up, I get redirected to this page: https://app.codegpt.co/agreements I receive the message "This page isn’t working".

**Comment by chuledeco** [comment]
I'm facing the same problem

**Comment by BurakKaraalp** [comment]
Same issue.

**Comment by aleparreira** [comment]
> Same issue. I tried in many browsers but I cannot log in.
> 
> Edit; I think I understand the cause of the problem. If you do not have a membership on the codegpt website, it does not direct you to the registration page. I created a membership by logging into the website from my mobile phone, and then when I logged in from my computer, the problem was solved.

Yes, I confirm that. Log in using mobile phone then worked reloading on computer worked.

**Comment by PilarHidalgo** [comment]
Hello @BurakKaraalp  we are reviewing the login procedure by mobile. We will write to you as soon as possible.

**Comment by PilarHidalgo** [comment]
@austinfarmer11 we have experienced some problems lately, but the web version of Studio is working now, please check again, clean the cookies and try!, Thanks in advance
